Src/VimCore/CoreInterfaces.fs

```diff
@@ -3644,6 +3647,119 @@ type BindDataStorage<'T> =
         | Simple bindData -> Simple (bindData.Convert mapFunc)
         | Complex func -> Complex (fun () -> func().Convert mapFunc)
 
+/// This is the result of attemping to bind a series of KeyInputData values
+/// into a Motion Command, etc ...

```

Comment:
See the discussion section of this PR for an explanation of the mapped bind family of types.

Reply:
I agree with the sentiment of your PR description here: this feels like it should be factorable. But I can't think of a clean way to do this. It's a small amount of repetition though so I don't feel like it's too bad. 


---
In reply to: [285844744](https://github.com/VsVim/VsVim/pull/2556#discussion_r285844744) [](ancestors = 285844744)